test-suite/libv2-demo/lib-gas-opt-pllel/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.25...3.29)
+project(demo_Wannier90 LANGUAGES Fortran)
```

Comment:
Here are functional tests for the library. Let me know what things to test in these cases.